qbittorrent/data/config/qBittorrent/qBittorrent.conf

```diff
@@ -0,0 +1,29 @@
+[AutoRun]
+enabled=false
+program=
+
+[BitTorrent]
+Session\DefaultSavePath=/downloads/complete
+Session\Port=6881
+Session\QueueingSystemEnabled=true
+Session\TempPath=/downloads/incomplete/
+
+[LegalNotice]
+Accepted=true
+
+[Meta]
+MigrationVersion=4
+
+[Network]
+Cookies=@Invalid()
+PortForwardingEnabled=false
+
+[Preferences]
+Connection\PortRangeMin=6881
+Connection\UPnP=false
+Downloads\SavePath=/downloads/complete
+Downloads\TempPath=/downloads/incomplete/
+WebUI\Address=*
+WebUI\Port=8094
+WebUI\ServerDomains=*
+WebUI\HostHeaderValidation=false
```

Comment:
If this is just the default `qBittorent.conf` then I think we can remove this as it will be generated when the app is installed. I see you have created this bind: `- ${APP_DATA_DIR}/data/config:/config` so it will be persisted across app restarts and updates.

If we can delete this file, then you can also delete the `qBittorrent` directory that the config file is saved in and, instead, place an empty `.gitkeep` file in `data/config` so that this empty directory is included in the repo.

Reply:
This config is needed as a workaround for an error that was preventing me from using the WebUI, and also to set the default download path to the "downloads/complete" folder instead of the "downloads" folder.

Reply:
pinging @lukechilds for his thoughts here as well. 

I see, thanks. Do you recall what the specific error was? I just tested removing this file and installing the app, and I can access the WebUI. 

A default `qBittorrent.conf` is generated automatically that looks like this, with the save path set to `/downloads/` instead of `/downloads/complete/` as you note.

```
[AutoRun]
enabled=false
program=

[BitTorrent]
Session\DefaultSavePath=/downloads/
Session\Port=6881
Session\QueueingSystemEnabled=true
Session\TempPath=/downloads/incomplete/

[LegalNotice]
Accepted=true

[Meta]
MigrationVersion=4

[Network]
PortForwardingEnabled=false

[Preferences]
Connection\PortRangeMin=6881
Connection\UPnP=false
Downloads\SavePath=/downloads/
Downloads\TempPath=/downloads/incomplete/
WebUI\Address=*
WebUI\Port=8094
WebUI\ServerDomains=*
```

Is the reason you want to set the downloads path to `/downloads/complete/` so that qBittorrent mimics Transmission's default behaviour? The other torrent app SimpleTorrent does not mimic this and instead saves files to `/downloads` and I'm assuming future torrent apps will have their own default behaviours.

I do actually like the idea of torrents being saved in `/downloads/complete/`, but I'm hesitant to include a `qBittorent.conf` in the repo because it will introduce complexity down the road if an app update requires us to change this file. Existing users who update their app won't get changes we make to this default file unless we add some finicky logic and we would need to be careful not to overwrite any custom changes users have made. Plus it is nice for the qBittorrent application itself to generate a config for us without any mistakes and then users can change the config from the UI (save to `/downloads/complete/' etc..).